src/project_types.rs

```diff
@@ -1,17 +1,38 @@
 /// Structure to represent a Proton Project.
 /// This is what will be written to a Protonfile at the project root.
-#[derive(Debug, RustcEncodable, RustcDecodable)]
+
```

Comment:
Get rid of this blank line. Comments should be directly next to what they're doc-ing. Otherwise it just looks weird. Do you not think so?